website/docs/terms/elt.md

```diff
@@ -0,0 +1,139 @@
+
+#### Promotes greater transparency for end busines users
+
+Data teams can expose the version-controlled code used to transform data for analytics to end business users by no longer having transformations hidden in the ETL process. Instead of having to manually respond to the common question, “How is this data generated?” data folks can direct business users to documentation and repositories. Having end business users involved or viewing the data transformations promote greater collaboration and awareness between business and data folks.
+
+## ELT tools
+
+As mentioned earlier, the recent development of certain technologies and products has helped lower the barrier of entry to implementing the ELT workflow. Most of these new products act as one or two parts of the ELT process, but some have crossover across all three parts. We’ll outline some of the current tools in the ELT ecosystem below.
+
+| Product | E, L, or T? | Description | Open-source option? |
+|---|---|---|---|
+| Fivetran/HVR | E, some T, L | Fivetran is a SaaS company that helps data teams extract, load, and perform some transformation on their data. Fivetran easily integrates with modern data warehouses and dbt. They also offer transformations that leverage dbt Core. | :x: |
+| Stitch by Talend | E, L | Stitch (part of Talend) is another SaaS product that has many data connectors to extract data and load it into data warehouses. | :x: |
+| Airbyte | E, L | Airbyte is an open-source and cloud service that allows teams to create data extraction and load pipelines. | :white_checl_mark: |
+| Funnel | E, some T, L | Funnel is another product that can extract and load data. Funnel’s data connectors are primarily focused around marketing data sources. | :x: |
+| dbt | T | dbt is the transformation tool that enables data analysts and engineers to transform, test, and document data in the cloud data warehouse. dbt offers both an open-source and cloud-based product. | :white_check_mark: |
+
+## Conclusion
+
+The past few years have been a whirlwind for the data world. The increased accessibility and affordability of cloud warehouses, no-code data extractors and loaders, and a true transformation layer with dbt has allowed for the ELT workflow to become the preferred analytics workflow. ETL predates ELT and differs in when data is transformed. In both processes, data is first extracted from different sources. However, in ELT processes, data is loaded into the target data platform and then transformed. The ELT workflow ultimately allows for data team members to extract, load, and model their own data in a flexible, accessible, and scalable way.
+
+## Further reading
+
+Check out some of our favorite content about the ELT workflow below:
+
+- [The Case for the ELT Workflow](https://www.getdbt.com/analytics-engineering/case-for-elt-workflow/)
+- [A Love Letter to ELT Tools](https://www.getdbt.com/analytics-engineering/etl-tools-a-love-letter/)
```

Comment:
This post is ETL tools btw

Reply:
Not following your comment here. Can you say that again in another way?

Reply:
The title of the actual post is "A Love Letter to ETL Tools" vs it's currently "A Love Letter to _ELT_ Tools"

Reply:
Ah I see now. I'm going blind after looking at these markdowns for so long lol. Thanks for pointing this out